### DISCUSSION TITLE
Translations are missing on the server

### DISCUSSION THREAD

**Discussion by ColinTimBarndt** [FabricMC/fabric-api#2384] [Mod Dev Support] [answered]
Hello, I've been working on a mod that adds commands which support translations, but I noticed that when running a server, the server console does not translate and behaves as if the translations were missing. This is not the case for the client. Is this a bug?

Example:
Mod: https://github.com/ColinTimBarndt/fabric_chat-emotes
- Player runs command `chat-emotes reload`.
- Player: `Reloaded emotes!`
- Server Console: `[Player: chat_emotes.commands.reload.success]`

**Comment by Linguardium** [comment, marked as answer]
servers dont load resource packs/assets.  the server jar has en_us.json inside it as the only asset and uses this for translations.  Even providing a url for a server resource pack wont load the resource pack, it only supplies the url to the clients.  The server does not have code to load resource packs/assets

**Reply by ColinTimBarndt** [reply]
Okay, so I need to use Mixins to provide my translations?

**Reply by ColinTimBarndt** [reply]
Also, does this mean that when I for example give myself modded items using commands, then the server console would only display the translation keys, right?

**Reply by Linguardium** [reply]
it should, yes. and yes. and also lets keep discussing on discord since we are in both places
